packages/eslint-plugin/src/rules/ban-tslint-comment.ts

```diff
@@ -0,0 +1,60 @@
+import { AST_TOKEN_TYPES } from '@typescript-eslint/experimental-utils';
+import * as util from '../util';
+
+// tslint regex
+// https://github.com/palantir/tslint/blob/95d9d958833fd9dc0002d18cbe34db20d0fbf437/src/enableDisableRules.ts#L32
+const ENABLE_DISABLE_REGEX = /^\s*tslint:(enable|disable)(?:-(line|next-line))?(:|\s|$)/;
+
+const toText = (
+  text: string,
+  type: AST_TOKEN_TYPES.Line | AST_TOKEN_TYPES.Block,
+): string =>
+  type === AST_TOKEN_TYPES.Line
+    ? ['//', text.trim()].join(' ')
+    : ['/*', text.trim(), '*/'].join(' ');
+
+export default util.createRule({
+  name: 'ban-tslint-comment',
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Bans `// tslint:<rule-flag>` comments from being used',
+      category: 'Stylistic Issues',
+      recommended: 'warn',
```

Comment:
cannot add a recommended rule without a major change
```suggestion
      recommended: false,
```